README.md

```diff
@@ -273,13 +273,18 @@ You can specify the browser and Node.js versions by queries (case insensitive):
     to PhantomJS runtime.
 * `extends browserslist-config-mycompany`: take queries from
   `browserslist-config-mycompany` npm package.
-* `supports es6-module`: browsers with support for specific features.
+* `fully supports es6-module`: browsers with full support for specific features.
   `es6-module` here is the `feat` parameter at the URL of the [Can I Use]
-  page. A list of all available features can be found at
-  [`caniuse-lite/data/features`].
+  page. For example `fully supports css-grid` will omit Edge 12-15, as those browser versions are marked as [having partial support].
+  A list of all available features can be found at [`caniuse-lite/data/features`].
+* `partially supports es6-module`: browsers with full or partial support for specific features.
```

Comment:
We need to mention old query name too

Reply:
The next bullet point in this list mentions `supports es6-module`. What if I remove the dedicated line below and update this bullet point to be:

> `partially supports es6-module` or `supports es6-module`: browsers with full or partial support for specific features. 

Does that sound good?

Reply:
Yes, sounds good